client/components/clipboard-button-input/index.jsx

```diff
@@ -84,4 +87,10 @@ class ClipboardButtonInputExport extends React.Component {
 	}
 }
 
-export default localize( ClipboardButtonInputExport );
+const wpcomEditorToolbarWebPreviewClipboardUrlButtonClick = () => {
```

Comment:
No need for external definitions and `composeAnalytics`. We use that only if you have multiple events. :)